osd/pull_secret_user_mismatch.json

```diff
@@ -2,7 +2,7 @@
   "severity": "Error",
   "service_name": "SREManualAction",
   "summary": "Action required: Review pull secret",
-  "description": "Your cluster requires you to take action because the Red Hat Site Reliability Engineering (SRE) have detected that your cluster's pull secret has been updated or modified by a user on your cluster. The user in the updated pull-secret doesn't match the cluster owner. Please fix the pull-secret, as the current configuration impacts Red Hat SRE's ability to monitor and operate your cluster, and your cluster's ability to initiate any upgrades.",
+  "description": "Your cluster requires you to take action because the Red Hat Site Reliability Engineering (SRE) have detected that your cluster's pull secret has been updated or modified by a user on your cluster. The user in the updated pull-secret doesn't match the cluster owner. Please fix the pull-secret, as the current configuration impacts Red Hat SRE's ability to monitor and operate your cluster, and your cluster's ability to initiate any upgrades. Please review the following documentation for more information: https://access.redhat.com/solutions/5899701",
```

Comment:
This is probably fine for right now, but I wonder what our guidance is on putting docs links into `doc_references`?